sklearn/preprocessing/_encoders.py

```diff
@@ -731,6 +765,14 @@ def inverse_transform(self, X):
 
         for i in range(n_features):
             labels = X[:, i].astype('int64', copy=False)
-            X_tr[:, i] = self.categories_[i][labels]
+            # set unknown values to None
+            unknown_labels = labels == self.unknown_value
```

Comment:
This can be moved into the `if` statement, since it is not needed with when `self.handle_unknown='error'`